### PR TITLE
Author schema changes

### PR DESCRIPTION
**Pull request by michamos**
I worked on this some time ago with @annetteholtkamp, but I never pushed because it is not strictly speaking final as we want to have user feedback about what there should be in author profile pages.
It still contains some useful refactorings, with useless fields removed and a clearer organization, so it might serve as a starting point.

This should probably not be merged before gathering the requirements as it will require changes in doJSON and in the formdata to model, which we might not want to do twice.

### CODE REVIEW

**Comment by jacquerie** [comment]
IMO after https://github.com/inspirehep/inspire-dojson/pull/66 is

- [ ] reviewed
- [ ] merged
- [ ] integrated in `inspire-next`
- [ ] tested on all Journals

this should go in: it's better to have a semi-stable schema than not.

File: inspire_schemas/records/authors.yml [hunk 1]
```diff
@@ -155,70 +262,71 @@ properties:
...
-            type: string
-        minItems: 1
-        type: array
-        uniqueItems: true
+            :MARC: ``970__d``
+
+            Contains a reference to the record replacing the current one, if it
+            is marked as :ref:`deleted`.
+        title: Record replacing this one
     positions:
```
Comment (drjova) [inline inspire_schemas/records/authors.yml]: flatten the institution history object

Comment (michamos) [reply inspire_schemas/records/authors.yml]: done, it's not exactly the same though (`name` vs `institution`).

File: scripts/generate_example_records.js [hunk 2]
```diff
@@ -26,7 +26,6 @@ jsf.format('orcid', function(gen, schema){
     }
     return almost_orcid + ck
 });
-jsf.format('.+, .+', function(gen, schema){ return gen.randexp('^.+, .+$');});
```
Comment (drjova) [inline scripts/generate_example_records.js]: I didn't know we have a js for this :P

**Comment by david-caro** [comment]
Merging and bumping the major version so we can start working on top of it.

**Pull request merged**